### DISCUSSION TITLE
Display nested field in the `fieldDistribution` object on `/stats`

### DISCUSSION THREAD

**Discussion by Karman40** [meilisearch/product#573] [Feedback & Feature Proposal]
**DEMO DATA:**
```
{
  field1: string
  field2: string
  nestedField: {
    innerField1: string
    innerField2: string
    ...
  }
  ...
}
```

GET `/indexes/{index_uid}/stats`, the response includes the following fields in the fieldDistribution: `field1, field2, nestedField`
In this case, the nested fields are not included in the statistics.

**It would be nice if it could work somehow as follows:**
`field1, field2, nestedField, nestedField.innerField1, nestedField.innerField2`

First of all, this is a great help in the settings page, because i don't need to remember the scheme of which fields and which route they are accessible, but i could just check in a select, for example, what i want to make searchable.

In my oppinien this as a separate API endpoint.

**Comment by gmourier** [comment]
Hello @Karman40 👋

Indeed, I think you are right. It should represent the field distribution exhaustively.

I've added that to our pipeline, but I can't give you an ETA yet. I will reach you when it's planned.

Thank you!

**Reply by Karman40** [reply]
@gmourier Thank you for your feedback!

If I can help you in any way, feel free to contact me!
Unfortunately, I'm not qualified in Rust, but I'll be happy to send you my Javascript example code if needed.
